CHANGELOG.md

```diff
@@ -246,7 +295,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 - References to AWS keys and variables.
 
+<<<<<<< HEAD
```

Comment:
There are merge conflict dividers in here too. Thanks.

Reply:
should be fine now.